com.cassidyjames.butler.json

```diff
@@ -0,0 +1,25 @@
+{
+    "app-id" : "com.cassidyjames.butler",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "com.cassidyjames.butler",
+    "finish-args" : [
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--share=ipc",
+        "--share=network",
+        "--device=dri"
+    ],
+    "modules": [{
+        "name": "butler",
+        "buildsystem": "meson",
+        "config-opts": ["--buildtype=release"],
```

Comment:
I don't think this line is needed, flatpak-builder sets this by default.

Reply:
Please tackle this post-merge.

Reply:
it doesn't (I can't find this in the source code of flatpak-builder). It's the default most of the time, but it can be set differently. I have had the issue with DISTRHO-Ports where it built debug by default.